.github/workflows/rust.yml

```diff
@@ -41,26 +41,26 @@ jobs:
           toolchain: ${{ matrix.rust }}
 
       - name: cargo build (debug; default features)
-        run: cargo build
+        run: cargo build --locked
```

Comment:
I feel like this should be in the same commit as the one that added `Cargo.lock`?